src/schemas/json/haxelib.json

```diff
@@ -0,0 +1,93 @@
+{
+	"$schema": "http://json-schema.org/draft-04/schema#",
+	"description": "A haxelib project",
+	"type": "object",
+	"properties": {
+		"name": { "$ref": "#/definitions/projectName" },
+		"url": {
+			"description": "Project's website",
+			"type": "string",
+			"format": "uri"
+		},
+		"description": {
+			"description": "Short description of the project",
+			"type": "string"
+		},
+		"license": {
+			"description": "Open source license under which the project is licensed",
+			"enum": ["GPL", "LGPL", "BSD", "Public", "MIT", "Apache"]
+		},
+		"version": { "$ref": "#/definitions/semver" },
+		"classPath": {
+			"description": "Folder in the package which contains the source files for this project",
+			"type": "string"
+		},
+		"contributors": {
+			"description": "List of project contributors that are allowed to upload to haxelib",
+			"type": "array",
+			"items": { "$ref": "#/definitions/userName" },
+			"minItems": 1,
+			"uniqueItems": true
+		},
+		"tags": {
+			"description": "List of tags for easier finding the project on haxelib",
+			"type": "array",
+			"items": { "$ref": "#/definitions/haxelibTag" },
+			"uniqueItems": true
+		},
+		"dependencies": {
+			"type": "object",
+			"description": "Project's dependencies",
+			"patternProperties": {
+				"^[A-Za-z0-9_.-]{3,}$": { "$ref": "#/definitions/dependencyVersion" }
+			},
+			"additionalProperties": false
+		},
+		"releasenote": {
+			"description": "Short description of changes made in this version",
+			"type": "string"
+		}
+	},
+	"additionalProperties": false,
```

Comment:
Usually this is considered a bad practice. You are effectively limiting anyone from inheriting/subschema'ing their schema from this one. It could be valid in this case, just wanted to call it to your attention